pkg/sql/parser/sql.y

```diff
@@ -8869,10 +8858,10 @@ multiple_set_clause:
 // TO {<name> | CURRENT_USER | SESSION_USER}
 // %SeeAlso: DROP OWNED BY
 reassign_owned_by_stmt:
-  REASSIGN OWNED BY role_spec_list TO role_spec
+  REASSIGN OWNED BY name_list TO role_spec
```

Comment:
Can you look into normalizing role_spec correctly as well?

See:
https://github.com/cockroachdb/cockroach/issues/55396

and:
https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/parser/sql.y#L1319
https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/parser/sql.y#L2478

Can be a separate PR